python/tvm/relay/analysis/analysis.py

```diff
@@ -351,3 +353,42 @@ def search_fc_transpose(expr):
     """
     ret = _ffi_api.search_fc_transpose(expr)
     return ret
+
+
+def get_calibration_data(mod, data):
+    """Get the calibration data of a given relay graph
+
+    This pass use the graph runtime to get the calibration data of a module, which
```

Comment:
We may need to think of the semantic for the module with control flows, which has to use VM instead of graph runtime.

Reply:
```suggestion
    This pass uses the graph runtime to get the calibration data of a module, which
```

Per offline discussion, please mention that this pass only works for the graph without control flow.